libs/api-bundle/composer.json

```diff
@@ -20,7 +21,6 @@
         "phpunit/phpunit": "^9.5",
         "sempro/phpunit-pretty-print": "^1.4",
         "symfony/framework-bundle": "^6.1",
-        "symfony/security-bundle": "^6.1",
```

Comment:
Tohle byla kravina. Protoze TokenInterface implementuje `Symfony\Component\Security\Core\User\UserInterface`, takze to nemuze byt jako dev zavislost.